torch/testing/_internal/inductor_utils.py

```diff
@@ -137,3 +138,30 @@ def inner(fn):
 )
 
 IS_BIG_GPU = LazyVal(lambda: HAS_CUDA and is_big_gpu())
+
+def maybe_skip_size_asserts(op):
+    """
+    For certain ops, there meta and eager implementation returns differents
+    strides. This cause size/strides assert fail. Skip adding those
+    asserts for now.
+    """
+    if (
+        op.aten_name
+        in (
+            "fft_hfftn",
+            "fft_ihfftn",
+            "fft_fft",
+            "fft_ifft",
+            "fft_ihfft",
+            "fft_ihfft2",
+            "fft_rfft",
+            "fft_rfft2",
+            "fft_rfftn",
+            "linalg_eig",
+            "linalg_eigvals",
+        )
+        and "TORCHINDUCTOR_SIZE_ASSERTS" not in os.environ
+    ):
+        return torch._inductor.config.patch(size_asserts=False)
```

Comment:
If these are failing - I think we should add back in the ops for fake incorrect cc @ezyang 

https://github.com/pytorch/pytorch/pull/145080/files#diff-516af4522f749fe6a58ccc82976eca0101346c82ed0311110d81b50bc973e940R232-R233

Reply:
We should fix this issue.  Incorrect size asserts means inductor gets the strides wrong and will generate incorrect code.

Reply:
Most of these failures happen in CI but I can not repro locally. I remember there is one I can repro locally, will try that

Reply:
> could you try reverting this change and see if the test passes?
https://github.com/pytorch/pytorch/pull/145080/files#diff-516af4522f749fe6a58ccc82976eca0101346c82ed0311110d81b50bc973e940R232-R233

Does not help.

Here is a test failure that I can repro locally:


```
TORCHINDUCTOR_SIZE_ASSERTS=1 PYTORCH_OPINFO_SAMPLE_INPUT_INDEX=0 python test/inductor/test_torchinductor_opinfo.py TestInductorOpInfoCUDA.test_comprehensive_fft_hfftn_cuda_float16
```